testing/vcs/test_vcs_configurator_click_label.py

```diff
@@ -0,0 +1,63 @@
+import vcs
```

Comment:
Makes sure that the everything properly responds to clicking (used to make sure the logic for selecting an object works when I make changes to it in the future); both of the next two tests do pretty much the same thing, but with markers / text instead of labels.